src/main/java/com/mail/controllers/EmailController.java

```diff
@@ -0,0 +1,49 @@
+package com.mail.controllers;
+
+import com.mail.entities.TextMessage;
+import com.mail.service.TextMessageService;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Avoid import all from package.
With this you pass many classes into autocomplete feature of IDE and when you write something it will give you many false suggestions.